GUI-dev/src/app/processstore.service.ts

```diff
@@ -0,0 +1,21 @@
+import {Injectable} from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import {Observable} from 'rxjs/Observable';
+ 
+const httpOptions = {
+    headers: new HttpHeaders({ 'Content-Type': 'application/json' })
+};
+ 
+@Injectable()
+export class ProcessStoreService {
+
+    
+  restApi = window.location.protocol + "//" + window.location.hostname + ":10000/api";

```

Comment:
Why are you using the `window` object here?
Please do not use something like this as we have a central service which is providing all endpoints and the associated urls.